lib/starknet-jvm.md

```diff
@@ -577,7 +577,7 @@ fun main(args: Array<String>) {
     // Import a compiled contract
     val contractCode = Path.of("contract.json").readText()
     val casmCode = Path.of("contract.casm").readText()
-    val contractDefinition = Cairo1ContractDefinition(contractCode)
+    val contractDefinition = CairoContractDefinition(contractCode)
     val casmContractDefinition = CasmContractDefinition(casmCode)
     val nonce = account.getNonce().send()
 
```

Comment:
let's also check javademo, perhaps it would also need similar change

Reply:
No, currently we have Cairo0 in javademo.